ONSdigital/dp-frontend-search-controller
# Implement topic cache for f&t sub topic page work

### What
Implemented new topic cache for topics and subtopic data aggregation routes
[Trello Link](https://trello.com/c/c34y1UgY/2064-implement-topic-cache-for-ft-sub-topic-page-work)

### How to review

- Pull branch
- set EnableReworkedDataAggregationPages to 'true'
- run application and try accessing the topic and subtopic routes in your browser
- Ensure correct data is displayed and cache working.

### Who can review
Anyone
